app/models/feedback_message.rb

```diff
@@ -1,8 +1,7 @@
 class FeedbackMessage < ApplicationRecord
-  belongs_to :offender, foreign_key: "offender_id", class_name: "User", optional: true, inverse_of: :feedback_messages
-  belongs_to :reviewer, foreign_key: "reviewer_id", class_name: "User", optional: true, inverse_of: :feedback_messages
-  belongs_to :reporter, foreign_key: "reporter_id", class_name: "User", optional: true, inverse_of: :feedback_messages
-  belongs_to :affected, foreign_key: "affected_id", class_name: "User", optional: true, inverse_of: :feedback_messages
+  belongs_to :offender, foreign_key: "offender_id", class_name: "User", optional: true, inverse_of: :offender_feedback_messages
```

Comment:
Changed `inverse_of` options so that the associations would work properly.

Reply:
Ahh thanks, I could never figure this out 😓